rpc/src/blobstream.rs

```diff
@@ -0,0 +1,33 @@
+use bytes::Bytes;
+use celestia_types::MerkleProof;
+use jsonrpsee::proc_macros::rpc;
+use prost::bytes;
+
+// DataRootTupleRoot is the root of the merkle tree created
+// from a set of data root tuples.
+pub type DataRootTupleRoot = Bytes;
+
+// DataRootTupleInclusionProof is the binary merkle
+// inclusion proof of a height to a data commitment.
+pub type DataRootTupleInclusionProof = MerkleProof;
```

Comment:
```suggestion
/// DataRootTupleRoot is the root of the merkle tree created
/// from a set of data root tuples.
pub type DataRootTupleRoot = Bytes;

/// DataRootTupleInclusionProof is the binary merkle
/// inclusion proof of a height to a data commitment.
pub type DataRootTupleInclusionProof = MerkleProof;
```

you need 3`/` to make it a doc-comment